plc4j/drivers/ads/src/main/java/org/apache/plc4x/java/ads/protocol/AdsProtocolLogic.java

```diff
@@ -136,11 +136,11 @@ public void onConnect(ConversationContext<AmsTCPPacket> context) {
         // otherwise just mark the connection as completed instantly.
         setupAmsRouteFuture.whenComplete((unused, throwable) -> {
             if (!configuration.isLoadSymbolAndDataTypeTables()) {
-                future.completeExceptionally(new PlcConnectionException(
-                    "Lazy loading is generally planned, but not implemented yet. " +
-                        "If you are in need for this feature, please reach out to the community."));
-            }
-            //if (configuration.isLoadSymbolAndDataTypeTables()) {
+                context.fireConnected();
+//                future.completeExceptionally(new PlcConnectionException(
```

Comment:
is the outcommmented code here an accident?

Reply:
No ... it's a first step to make it possible to still connect to an ADS without loading the symbol table (if you just use raw addresses) ...